packages/matter.js/src/codec/BtpCodec.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ByteArray, Endian } from "../util/ByteArray.js";
+import { DataReader } from "../util/DataReader.js";
+import { DataWriter } from "../util/DataWriter.js";
+
+export interface HandshakeRequest {
```

Comment:
Because we export the names maybe better to make them a bit moreunique by prepending Btp to all of these? WDYT? ... so BtpHandshakeRequest ... samdé for the following names

Reply:
Sure, on it